README.md

```diff
@@ -117,36 +124,52 @@ The list of all Hack Sessions held by date:
 | 19/01/2018 | [Git: Beyond the Basics](https://www.meetup.com/devstaff/events/246863163/) |
 | 19/09/2017 | [Ansible Hack Session #1](https://www.meetup.com/devstaff/events/243440678/) |
 | 01/12/2016 | [Django Hack Session](https://www.meetup.com/devstaff/events/235617760/) |
-| 01/09/2016 | [Hashicorp Vault, AWS KMS, Chef/Ansible Vault, Consul](http://www.meetup.com/DevStaff-A-Developer-Community-Gathering-In-Crete/events/233637796/) |
+| 01/09/2016 | [Hashicorp Vault, AWS KMS, Chef/Ansible Vault, Consul](https://www.meetup.com/DevStaff-A-Developer-Community-Gathering-In-Crete/events/233637796/) |
 | 12/01/2016 | [Tor Hack Session](https://github.com/DaKnOb/TorConfig) |
 | 01/12/2015 | [Docker Hack Session](https://github.com/devstaff-crete/docker-hack-sessions) |
 
-More hack sessions have been organized, but they are missing from this list. If you know any, please help adding them by submitting a Pull Request.
+More hack sessions have been organized, but they are missing from this list. If
+you know any, please help adding them by submitting a Pull Request.
 
 ## Projects :construction:
 
-Find [Open Source Software](projects/README.md) and Projects currently developed by DevStaff Members. You can add your own by sending us a pull request.
+Find [Open Source Software](projects/README.md) and Projects currently
+developed by DevStaff Members. You can add your own by sending us a Pull
+Request.
 
 ## Job Postings :postal_horn:
 
-Check the [job openings](jobs/README.md), posted by DevStaff members. You can add your own by
-sending us a pull request.
+Check the [job openings](jobs/README.md), posted by DevStaff members. You can
+add your own by sending us a Pull Request.
 
 ## Org Meetups
 
-An org meetup is a simple get-together, **open to every member of our community** who would like to contribute / discuss some organizational issues, future topics etc. Also a good opportunity to chat about all-things-tech :)
+An org meetup is a simple get-together, **open to every member of our
+community**, who would like to contribute / discuss some organizational issues,
+future topics etc. Also a good opportunity to chat about all-things-tech :)
 
-DevStaff is an open community. Open, as in Open Source. If you don't like anything about the community you can change it! 
+DevStaff is an open community. Open, as in Open Source. If you don't like
+anything about the community you can change it!
```

Comment:
maybe we can rephrase this to "If you don't like `something` about the community you can change it"

Reply:
Done.